kolibri/core/assets/src/state/actions.js

```diff
@@ -200,10 +200,15 @@ function kolibriLogout(store) {
   }).catch(error => { handleApiError(store, error); });
 }
 
-function getCurrentSession(store) {
+function getCurrentSession(store, force = false) {
```

Comment:
@jonboiser Thoughts on adding this parameter? I added it so the session would be properly fetched after the profile is updated.